resources/benchmark/tpcds/tables/customer.csv.json

```diff
@@ -87,7 +87,7 @@
     },
     {
       "name": "c_last_review_date",
-      "type": "string",
+      "type": "int",
```

Comment:
Looking at the schema graphic (the SVG I sent you for the presentation), it states that `c_last_review_date` should be a char field (string in our case)?

Reply:
I got the types from third_party/tpcds-kit/tools/tpcds.sql, there it is an integer.